src/realtime.d.ts

```diff
@@ -68,7 +65,65 @@ export interface Entity {
   vehicle?: VehicleUpdate;
   is_deleted: boolean;
 }
-
+interface active_period {
+  start?: Date;
+  end?: Date;
+}
+interface TripDescriptor {
+  trip_id: string;
+  start_time?: string;
+  end_time?: string;
+}
+interface informed_entity_route {
```

Comment:
could we use PascalCase for the names of interfaces please, just to be consistent with the existing interfaces. 

e.g. `informed_entity_route` -> `InformedEntityRoute`